R/03-stylePlot.R

```diff
@@ -426,7 +426,7 @@ stylePlot <- function(input, output, session, savedData) {
 }
 
 getLastSelected <- function(lastSelected, default) {
-  if_else(is.null(lastSelected),
+  ifelse(is.null(lastSelected),
```

Comment:
Could you remove `if_else` also from the namespace since it is not used at all anymore?